### PR TITLE
Adds initial ROADMAP.md including a minimal Q2-2019 roadmap

### PR DESCRIPTION
**Pull request by ellis-bigelow**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/3)

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# KF Serving 2019 Roadmap
+## Q2 2019
+
+### Core CUJs
+* High Level Interfaces
```
Comment (abhi-g) [inline ROADMAP.md]: Can you please add a couple sentences outlining the motivation for each high level CUJ here.

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    * Deploy a Tensorflow model without specifying a Tensorflow Serving Technology.
+    * Deploy a XGBoost model without specifying a XGBoost Serving Technology.
+    * Deploy a ScikitLearn model without specifying a ScikitLearn Serving Technology.
+    * Deploy a Custom Containerized model by specifying.
+
+* Model Rollout
```
Comment (abhi-g) [inline ROADMAP.md]: eg. for this one could be: Create a minimal support for experimenting and rolling a new model to an existing production deployment.

File: ROADMAP.md (hunk 1 above)
Comment (ellis-bigelow) [reply ROADMAP.md]: ack

File: ROADMAP.md (hunk 2 above)
Comment (ellis-bigelow) [reply ROADMAP.md]: ack

File: README.md [hunk 3]
```diff
@@ -1,2 +1,6 @@
 # kfserving
-Model serving related infrastructure in Kubeflow
+KF Serving provides a Custom Resource Definition for serving ML Models on arbitrary frameworks. It aim at solving 80% of model serving use cases by providing performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and Custom containers.
```
Comment (abhi-g) [inline README.md]: s/aim/aims/

**Comment by abhi-g** [comment]
/lgtm
please fix the typo mentioned above, and then its good to go. Since you are one of the approvers, you can self-approve :)

File: ROADMAP.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# KF Serving 2019 Roadmap
+## Q2 2019
+
+### Core CUJs
+Objective: "Simplify the user experience and provide a low barrier to entry by minimizing the amount of YAML necessary to deploy a trained model."
+* High Level Interfaces
+    * Deploy a Tensorflow model without specifying a Tensorflow Serving Technology.
+    * Deploy a XGBoost model without specifying a XGBoost Serving Technology.
+    * Deploy a ScikitLearn model without specifying a ScikitLearn Serving Technology.
```
Comment (yuzisun) [inline ROADMAP.md]: Did we miss PyTorch here?

Comment (ellis-bigelow) [reply ROADMAP.md]: Fixed

**Pull request merged**